lib/code_corps/github/sync/issue/issue.ex

```diff
@@ -1,45 +1,38 @@
 defmodule CodeCorps.GitHub.Sync.Issue do
-  alias CodeCorps.{GitHub, GithubIssue, GithubPullRequest, GithubRepo}
-  alias GitHub.Sync.Issue.GithubIssue, as: IssueGithubIssueSyncer
-  alias GitHub.Sync.Issue.Task, as: IssueTaskSyncer
-  alias GitHub.Sync.User.RecordLinker, as: UserRecordLinker
+  alias CodeCorps.{
+    GithubPullRequest,
+    GithubRepo,
+    GitHub.Sync
+  }
   alias Ecto.Multi
 
   @doc ~S"""
-  Syncs a GitHub issue API payload with our data.
+  Performs sync of a github issue payload related to a repo and a pull request.
 
-  The process is as follows:
-
-  - match with `CodeCorps.User` using `CodeCorps.GitHub.Sync.User.RecordLinker`
-  - create or update the `CodeCorps.Task` for the `CodeCorps.Project` in the
-    matched `CodeCorps.GithubRepo`
-
-  If the sync succeeds, it will return an `:ok` tuple with the created or
-  updated task.
-
-  If the sync fails, it will return an `:error` tuple, where the second element
-  is the atom indicating a reason.
+  Performs work identical to `sync/2`, with the adition of associating the
+  resulting `CodeCorps.GithubIssue` with the specified `CodeCorps.GithubPullRequest`.
```

Comment:
I still find that I'm forced to use the full module path for generated doc html references to work correctly.